transit_odp/timetables/extract.py

```diff
@@ -137,14 +139,52 @@ def extract(self) -> ExtractedData:
             columns=["file_id", "route_link_ref", "from_stop_ref", "to_stop_ref"]
         ).set_index(["file_id", "route_link_ref"])
 
+        # extract flexible journey patterns
+        logger.debug("Extracting flexible journey patterns")
+        flexible_journey_patterns = self.extract_flexible_journey_patterns()
```

Comment:
at the moment `journey_patterns` is also storing flexible journey patterns. need to remove the redundant rows from one of these dataframes

Reply:
removed